### ISSUE TITLE
When getting numeric value with ","  grafana cant readt the numeric value 

### ISSUE THREAD

**Issue by yossiv** [neuralfraud/grafana-prtg#60]
Hi 
graph was working on version 3 .1.0
for some reason on the new plugin version once i want to pull the  Transfer rate value 
it givs me an error 
![image](https://user-images.githubusercontent.com/19520218/30498298-cd924442-9a5e-11e7-95c5-0bbe367b64a4.png)

this is how it looks like on the PRTG  
the transfer rate value coming with  "," 
![image](https://user-images.githubusercontent.com/19520218/30498363-1b818e6a-9a5f-11e7-8978-2bc9e182dd89.png)

**Comment by neuralfraud** [comment]
I don't believe I've come across a sensor with this type of data output. What sensor is it? Provide the output from the "histdata.xml" API request that contains the information and I'll see what can be done about it.

**Comment by yossiv** [comment]
Hi @neuralfraud  i made some more investigation i found that when im increasin the time more than 12 hours the Graph show the data.
if it less then 12 hours its show no data. 

![image](https://user-images.githubusercontent.com/19520218/32142571-ba0c1da6-bca2-11e7-9665-e5e6b52faad9.png)
more than 12 hours scale.
![image](https://user-images.githubusercontent.com/19520218/32142576-caeb4d9a-bca2-11e7-8e94-d5c2473bef1c.png)
